prometheus_client/metrics.py

```diff
@@ -186,6 +186,11 @@ def remove(self, *labelvalues):
         with self._lock:
             del self._metrics[labelvalues]
 
+    def reset(self):
```

Comment:
What would you think of naming this `remove_all` instead of `reset` to match `remove`? I could imagine `reset` also meaning something like "reset all values to zero" or something else.

Reply:
Actually, according to https://prometheus.io/docs/instrumenting/writing_clientlibs/#labels, I think it should be called `clear()`.

Reply:
You're right, I prefer `remove_all` than `reset` too.